aws/scripts/setup-kind-cluster.sh

```diff
@@ -2,7 +2,33 @@
 set -xe
 
 # Boot kind cluster
-nvkind cluster list|grep 'No kind clusters found.' && { nvkind cluster create; sleep 5; }
+cat <<EOF > kind-cluster.yaml
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+  kubeadmConfigPatches:
+  - |
+    kind: InitConfiguration
+    nodeRegistration:
+      kubeletExtraArgs:
+        node-labels: "ingress-ready=true"
+  extraPortMappings:
+  - containerPort: 80
+    hostPort: 80
+    protocol: TCP
+  - containerPort: 443
+    hostPort: 443
+    protocol: TCP
+- role: worker
+  extraMounts:
+    # We inject all NVIDIA GPUs using the nvidia-container-runtime.
+    # This requires `accept-nvidia-visible-devices-as-volume-mounts = true` be set
+    # in `/etc/nvidia-container-runtime/config.toml`
+    - hostPath: /dev/null
+      containerPath: /var/run/nvidia-container-devices/0
```

Comment:
so no gpu is assigned to the worker?

Reply:
actually let me fix to follow your template.

Reply:
done